web/src/main/java/se/inera/intyg/intygstjanst/web/service/repo/model/CitizenCertificateConverterImpl.java

```diff
@@ -62,13 +62,18 @@ public CitizenCertificate convert(Certificate certificate, List<Relation> relati
     private List<CitizenCertificateRelationDTO> getRelations(String certificateId, List<Relation> relations) {
         return relations
                 .stream()
-                .filter((relation) -> relation.getRelationKod().equals(RelationKod.ERSATT.toString()))
+                .filter(this::isRelationIncluded)
```

Comment:
Suggest that you remove the filter (and isRelationIncluded) from this implementation and let the CitizenCertificateRelationConverter do all the work. 

The converter already returns a empty Optional when a relation isn't mapped for some reason and it seems redundant to have filter logic in both places.